app/src/pages/NodesPage/EditNodeForm/EditNodeForm.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react';
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+
+import { fieldsContainer } from './styled';
+import { IEditNodeProps } from './interfaces';
```

Comment:
types.ts contains types and interfaces
```suggestion
import { IEditNodeProps } from './types';
```

Reply:
fix all over the app